common/supplemental/supplementalData.xml

```diff
@@ -4787,91 +4787,91 @@ XXX Code for transations where no currency is involved
+              <era type="0" start="-3760-10-7" code="am"/> <!-- Anno Mundi -->
           </eras>
         </calendar>
         <calendar type="buddhist">
           <calendarSystem type="solar" />
           <eras>
-              <era type="0" start="-542-01-01" code="buddhist" aliases="be"/>
+              <era type="0" start="-542-01-01" code="be"/> <!-- Buddhist Era -->
           </eras>
         </calendar>
         <calendar type="coptic">
           <calendarSystem type="other"/>
           <eras>
-              <era type="0" end="284-08-28" code="coptic-inverse"/>
-              <era type="1" start="284-08-29" code="coptic"/>
+              <era type="0" end="284-08-28" code="bd"/> <!-- Before Diocletian. NOTE: No evidence this era is used -->
+              <era type="1" start="284-08-29" code="am" primary="true"/> <!-- Anno Martyrum -->
           </eras>
         </calendar>
         <calendar type="persian">
           <calendarSystem type="solar"/>
           <eras>
-              <era type="0" start="622-01-01" code="persian" aliases="ap"/>
+              <era type="0" start="622-01-01" code="ap" aliases="sh hs"/> <!-- Anno Persico -->
           </eras>
         </calendar>
         <calendar type="dangi">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-2332-01-01" code="dangi"/> <!-- 'sequential' year -->
+              <era type="0" start="-2332-01-01"/> <!-- 'sequential' year -->
           </eras>
         </calendar>
         <calendar type="ethiopic">
           <calendarSystem type="other"/>
           <eras>
-              <era type="0" end="8-08-28" code="ethioaa" aliases="ethiopic-amete-alem mundi"/>
-              <era type="1" start="8-08-29" code="ethiopic" aliases="incar"/>
+              <era type="0" end="8-08-28" code="aa" aliases="mundi"/> <!-- Amete Alem -->
+              <era type="1" start="8-08-29" code="am" aliases="incar" primary="true"/> <!-- Amete Mihret -->
           </eras>
         </calendar>
         <calendar type="ethiopic-amete-alem">
           <eras>
-              <era type="0" end="-5492-08-29" code="ethioaa" aliases="ethiopic-amete-alem mundi"/>
+              <era type="0" end="-5492-08-29" code="aa" aliases="mundi"/> <!-- Amete Alem -->
           </eras>
         </calendar>
         <calendar type="indian">
           <eras>
-              <era type="0" start="79-01-01" code="indian" aliases="saka"/>
+              <era type="0" start="79-01-01" code="saka"/>
           </eras>
         </calendar>
         <calendar type="roc">
           <eras>
-              <era type="0" end="1911-12-31" code="roc-inverse" aliases="before-roc"/>
-              <era type="1" start="1912-01-01" code="roc" aliases="minguo"/>
+              <era type="0" end="1911-12-31" code="minguo-qian" aliases="broc before-roc"/>
+              <era type="1" start="1912-01-01" code="minguo" primary="true"/>
```

Comment:
this page https://en.wikipedia.org/wiki/Chinese_calendar_correspondence_table uses "AM", presumably that means "Anno Minguoae" (pardon my Latin).

Reply:
oh my